project/CLI.py

```diff
@@ -92,15 +105,41 @@ def __create(self, args):
             return "Game Created"
         return "Game Failed"
 
+    def __performTeamEdit(self, args):
+        if args[1] == "name":
+            if self.game.modify_team(args[2], args[4]):
+                return "Team name changed"
+        elif args[1] == "password":
+            if self.game.modify_team(args[2], args[4]):
+                return "Team password changed"
+        else:
+            return "Invalid command, check 2nd argument"
+
+    #bit complicated because both GM and teams can edit (their own for team), (GM -> any team)
+    def __editteam(self, args):
+        try:
```

Comment:
I think this can be factored out into a decorator (need_self or something like that). I'll look into it.

Reply:
Alright, that'll clean up this method if you can figure it out.